ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/install-on-bastionhost/provision/cucushift-installer-rehearse-gcp-ipi-install-on-bastionhost-provision-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-install-on-bastionhost-provision
+  steps: 
+  - ref: gcp-provision-vpc
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - chain: ipi-conf-gcp
+  - ref: ipi-conf-manual-creds
+  - ref: ipi-conf-gcp-oidc-creds-provision
+  - ref: ipi-conf-manual-creds-remove-unnecessary-creds
```

Comment:
Suggest to remove this step, add `EXTRACT_MANIFEST_INCLUDED: "true"` env on job config level, we are going to drop `ipi-conf-manual-creds-remove-unnecessary-creds`.